frontend/redux/nodes/entities/hosts/helpers.js

```diff
@@ -4,14 +4,20 @@ export const parseEntityFunc = (host) => {
   const { network_interfaces: networkInterfaces } = host;
   const networkInterface = networkInterfaces && find(networkInterfaces, { id: host.primary_ip_id });
 
-  let clockSpeed = null;
-  let clockSpeedFlt = null;
   let hostCpuOutput = null;
-
-  if (host && host.cpu_brand) {
-    clockSpeed = host.cpu_brand.split('@ ')[1] || host.cpu_brand.split('@')[1];
-    clockSpeedFlt = parseFloat(clockSpeed.split('GHz')[0].trim());
-    hostCpuOutput = `${host.cpu_physical_cores} x ${Math.floor(clockSpeedFlt * 10) / 10} GHz`;
+  if (host) {
+    let clockSpeedOutput = null;
+    try {
+      const clockSpeed = host.cpu_brand.split('@ ')[1] || host.cpu_brand.split('@')[1];
+      const clockSpeedFlt = parseFloat(clockSpeed.split('GHz')[0].trim());
+      clockSpeedOutput = Math.floor(clockSpeedFlt * 10) / 10;
+    } catch (e) {
+      // Some CPU brand strings do not fit this format and we can't parse the
```

Comment:
Should there be a console.log(e) here?

Reply:
I had debated adding a console log, but thought it may end up fairly noisy. We probably should add it and 99% of users won't notice the noise.

Reply:
This is done.